icons/cricket.svg

```diff
@@ -0,0 +1 @@
+<svg aria-labelledby="simpleicons-cricket-icon" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title id="simpleicons-cricket-icon">Cricket icon</title><path d="M8.198 9.994L3 15.192V0h5.198v9.994z"/><path fill-rule="nonzero" d="M13.657 5.978L3 16.564v7.349L20.982 5.978z"/><path d="M13.1 15.206l7.858 8.14h-7.325l-4.224-4.449 3.692-3.691z"/></svg>
```

Comment:
Can you provide a source that displays the Circket icon like this? Although I like the design I don't feel like it adheres to the company guidelines. However, I'm unfamiliar with the brand and struggled to find official guidelines...

Reply:
I made an adjustment to the mark to make it work in one-color. I'll remove it for now until I can find official documentation.